### PR TITLE
Instrumentation and cleanup

### PR DESCRIPTION
**Pull request by k45514n**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,57 +20,65 @@ function encrypt (data, key) {
...
   var pathJwt = event.pathParameters.token;
   jsonwebtoken.verify(pathJwt, keys.public,
     { algorithms: [ 'RS256' ] },
     (err, decodedJwt) => {
-      if (err) return callback(null, {
-        "statusCode": 400,
-        "body": `Error in pathJwt: ${err}\n`
-      });
+      if (err) {
+        context.iopipe.metric('jwt-error', true);
```
Comment (ewindisch) [inline index.js]: either log or metric! Metric is the new name for log

Comment (ewindisch) [reply index.js]: (here you're making two custom metrics of the same name)

File: README.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ sudo npm i -g serverless
 
 ### Deploy via `sls`
 
+CHANGE THE BUCKET NAME
```
Comment (ewindisch) [inline README.md]: better description of what you intend here in README

File: index.js [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
+const iopipe = require('@iopipe/iopipe');
 const aws = require('aws-sdk');
 const jsonwebtoken = require('jsonwebtoken');
 const fs = require('fs');
 const keys = require('./keys');
 const crypto = require('crypto');
 
-//const iopipe_profiler = require('@iopipe/profiler');
 const IOpipe = iopipe({
   token:"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJhNDAyZTQzNy0wNzIyLTQ0ZDktOGUyNy1jMGFjMjc2MzgxZTQiLCJqdGkiOiJmYTQzMDQxMi0zYThkLTQ5MzgtODA4My00YWIxODRjNmZmZjEiLCJpYXQiOjE1MDAzODgzMjgsImlzcyI6Imh0dHBzOi8vaW9waXBlLmNvbSIsImF1ZCI6Imh0dHBzOi8vaW9waXBlLmNvbSxodHRwczovL21ldHJpY3MtYXBpLmlvcGlwZS5jb20vZXZlbnQvLGh0dHBzOi8vZ3JhcGhxbC5pb3BpcGUuY29tIn0.Iez7L1pRsC1gk50H6-Qh99ZaduFfCixAPxgkfPmpElI",
```
Comment (ewindisch) [inline index.js]: we need to remove this token before release and include docs on how to add the token and signup for IOpipe

**Review by ewindisch — DISMISSED** [review]

File: index.js (hunk 1 above)
Comment (k45514n) [reply index.js]: Got it will fix, thanks!

File: README.md (hunk 2 above)
Comment (k45514n) [reply README.md]: GAH how did I miss that, thankees!

File: index.js (hunk 3 above)
Comment (k45514n) [reply index.js]: Agreed.

File: index.js [hunk 4]
```diff
@@ -1,16 +1,12 @@
...
 const jsonwebtoken = require('jsonwebtoken');
 const fs = require('fs');
 const keys = require('./keys');
 const crypto = require('crypto');
 
-//const iopipe_profiler = require('@iopipe/profiler');
 const IOpipe = iopipe({
-  token:"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJhNDAyZTQzNy0wNzIyLTQ0ZDktOGUyNy1jMGFjMjc2MzgxZTQiLCJqdGkiOiJmYTQzMDQxMi0zYThkLTQ5MzgtODA4My00YWIxODRjNmZmZjEiLCJpYXQiOjE1MDAzODgzMjgsImlzcyI6Imh0dHBzOi8vaW9waXBlLmNvbSIsImF1ZCI6Imh0dHBzOi8vaW9waXBlLmNvbSxodHRwczovL21ldHJpY3MtYXBpLmlvcGlwZS5jb20vZXZlbnQvLGh0dHBzOi8vZ3JhcGhxbC5pb3BpcGUuY29tIn0.Iez7L1pRsC1gk50H6-Qh99ZaduFfCixAPxgkfPmpElI",
-  /*plugins: [ iopipe_profiler({ enabled: true }) ]*/
+  token:"YOUR_IOPIPE_ACCESS_TOKEN",
```
Comment (ewindisch) [inline index.js]: could we just remove this and put it into the serverless.yml instead? (or even into a separate .yml or .json file? --serverless.yml can source external files)

File: index.js [hunk 5]
```diff
@@ -24,57 +20,64 @@ function encrypt (data, key) {
...
+          "body": `Error in pathJwt: ${err}\n`
+        });
+      }
 
       var hash = crypto.createHash('sha256');
       hash.update(pathJwt);
       var fileName = `${hash.digest('hex')}.json`;
 
       /* We've received a user's request, encrypt and "bin" it!  */
+      context.iopipe.label('recieved-user-request');
```
Comment (ewindisch) [inline index.js]: not sure this is a useful label. Lets can it?

**Review by ewindisch — DISMISSED** [review]

**Comment by coreylight** [comment]
@nodebotanist @ewindisch bump to close or merge

**Review by ewindisch — APPROVED** [review]

**Pull request merged**